vllm/core/evictor_v2.py

```diff
@@ -109,6 +109,7 @@ def add(self, block_id: int, content_hash: int, num_hashed_tokens: int,
 
     def update(self, block_id: int, last_accessed: float):
         self.free_table[block_id].last_accessed = last_accessed
```

Comment:
Maybe add a comment here: put newest element to the end of free_table to make sure free_table is sorted

Reply:
ok,I already added the comment